siliconcompiler/core.py

```diff
@@ -1612,10 +1615,11 @@ def check_manifest(self):
 
         # Dynamic checks
         # We only perform these if arg, step and arg, index are set.
-        # We don't check inputs for checkonly pass through
+        # We don't check inputs for skip all
+        # TODO: Need to add skip step
```

Comment:
Mark WIP in schema if not implemented yet?

Reply:
Until we start getting bug reports stating that someone has used an unimplemented feature and failed, I don't think we should spend too much time worrying about this. The schema defines the spec, not the implementation.